CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ---
 
+## Master
+
+### Bug fixes
+
+* Fixed using filter expression in call node.
```

Comment:
Missing two spaces at the end of the line after the `.` — for markdown formatting purposes